docs/cimpl6LanguageReference.md

```diff
@@ -339,27 +319,36 @@ There are restrictions on which building blocks can inherit from other building
 | `Entry` | `Abstract` or `Entry` |
 | `Abstract` | `Abstract` |
 
->**Note:** Although the CIMPL 6.0 tooling does not currently enforce these inheritance rules, they will be enforced in future releases. You are strongly encouraged to comply with these rules, and refactor any definitions ported from CIMPL 5.x that do not comply.
+Examples:
 
-Example:
+```
+Entry:     BodyWeight
+Parent:    VitalSign
+```
 
- `Entry:  WhiteBloodCellCountObservation`
+```
+Group:     Age
+Parent:    Quantity
+```
 
- `Parent: LaboratoryObservation`
+```
+Element:   EncounterParticipant
+Parent:    Participant
+```
 
 ### Path
 
-The `Path` keyword allows for abbreviations of long URLs. This functionality is optional; it is provided to make authoring easier. By convention, the alias should be be an all UPPER CASE word, although the tooling allows for numbers and hyphens after the first letter.
+The `Path` keyword allows for abbreviations of long URLs. This functionality is optional; it is provided to make authoring easier. By convention, the alias should be UPPERCASE, although the tooling allows for numbers and hyphens after the first letter.
 
 Example:
 
-`Path: FHIR = http://hl7.org/fhir/ValueSet`
+`Path: FHIRVS = http://hl7.org/fhir/ValueSet`
 
 ### Property
 
-Classes based on `Group`, `Entry`, and `Abstract` are composed of one or more properties (called _fields_ or _attributes_ in object-oriented programming). A property can be any `Element`, `Group`, or `Entry`, but never a primitive data type. Each `Property` must have a specified cardinality range, represented as `min..max`, indicating the number of repeats of the `Property`. The `Property` keyword cannot be used in `Element`.**
+Classes based on `Group`, `Entry`, and `Abstract` are composed of one or more properties (called _fields_ or _attributes_ in object-oriented programming). A property can be any `Element`, `Group`, or `Entry`, but never a primitive data type. Each `Property` must have a specified cardinality range, represented as `min..max`, indicating the number of repeats of the `Property`. The `Property` keyword cannot be used in the definition of an `Element`.
 
-In the following example, _StudyArm_ has four properties: _Name_, _Type_, _Comment_ and _ResearchStudy_. _Name_ is singular and required, _Type_ is optional and repeating, and _Comment_ is singular and optional:
+In the following example, _StudyArm_ has four properties: _Name_, _Type_, _Comment_ and _ResearchStudy_. _Name_ and _ResearchStudy_ are singular and required, _Type_ is optional and repeating, and _Comment_ is singular and optional:
```

Comment:
Since the references to Name, Type, Comment and ResearchStudy are in code blocks in the example, they should also be in code blocks in the text.

Reply:
Quite right. Fixed.